script.js

```diff
@@ -1,4 +1,176 @@
 var main = function (input) {
-  var myOutputValue = 'hello world';
```

Comment:
the first (var myOutputValue = 'hello world') is not used so we dont have to keep it inside.
you can declare myOutputValue with var myOutputValue ; without assigning a value first in this case.